apps/meteor/client/views/admin/settings/MemoizedSetting.tsx

```diff
@@ -77,32 +81,35 @@ const MemoizedSetting = ({
 	invisible = undefined,
 	...inputProps
 }: MemoizedSettingProps): ReactElement | null => {
+	const t = useTranslation();
 	if (invisible) {
 		return null;
 	}
 
 	const InputComponent = inputsByType[type];
 
 	return (
-		<Field className={className}>
-			<InputComponent
-				value={value}
-				editor={editor}
-				onChangeValue={onChangeValue}
-				onChangeEditor={onChangeEditor}
-				{...inputProps}
-				disabled={disabled}
-			/>
-			{hint && <Field.Hint>{hint}</Field.Hint>}
-			{callout && (
-				<Margins block='x16'>
-					<Callout type='warning'>{callout}</Callout>
-				</Margins>
-			)}
+		<Field className={className} flexDirection='row' justifyContent='space-between' alignItems='flex-start'>
+			<Box flexDirection='column' flexGrow={1}>
+				<InputComponent
+					value={value}
+					editor={editor}
+					onChangeValue={onChangeValue}
+					onChangeEditor={onChangeEditor}
+					{...inputProps}
+					disabled={disabled}
+				/>
+				{hint && <Field.Hint>{hint}</Field.Hint>}
+				{callout && (
+					<Margins block='x16'>
+						<Callout type='warning'>{callout}</Callout>
+					</Margins>
+				)}
+			</Box>
 			{enterpriseCallout && (
```

Comment:
Since you are not displaying the `enterpriseCallout` as a ReactNode and it's used only for checking if it's not an enterprise edition I suggest removing the `enterpriseCallout` declaration in the `Setting` component and renaming the prop to something more appropriate like: `showUpgradeButton`